core/src/main/res/values/strings.xml

```diff
@@ -730,6 +728,15 @@
     <string name="sort_duration_short_long">Duration (Short \u2192 Long)</string>
     <string name="sort_duration_long_short">Duration (Long \u2192 Short)</string>
 
+    <string name="sort_q_title_a_z">A \u2192 Z</string>
+    <string name="sort_q_title_z_a">Z \u2192 A</string>
+    <string name="sort_q_date_new_old">New \u2192 Old</string>
+    <string name="sort_q_date_old_new">Old \u2192 New</string>
+    <string name="sort_q_duration_short_long">Short \u2192 Long</string>
+    <string name="sort_q_duration_long_short">Long \u2192 Short</string>
+    <string name="sort_q_episode_title_a_z">A \u2192 Z</string>
+    <string name="sort_q_episode_title_z_a">Z \u2192 A</string>
```

Comment:
I think it would be better without the `q_`. Then the strings can be re-used in other locations (currently, there are no other locations, though)

Reply:
Actually these strings are used here for the feed page
https://github.com/AntennaPod/AntennaPod/blob/bcea11b6112ca67f92ebbcb3ba08cbf9b8a3a130/core/src/main/res/values/strings.xml#L724

I cannot reuse the strings from those because they put the thing that we are sorting into the string.

https://user-images.githubusercontent.com/149837/78513205-d6a52600-775e-11ea-9cc1-66d8093b597a.png